### ISSUE TITLE
IC 1 level shifter

### ISSUE THREAD

**Issue by Pintglass** [arananet/TzxDuino-Reloaded#9]
This is more of an observation than an issue, you have the level shifting IC  74HC4050 connected to the 5V rail, is this correct as for this IC to convert to 3.3V for the SD card shouldn't  pin 1 VCC of IC 1 be connected to the 3.3V rail?

**Comment by zbrne** [comment]
Yes, this is indeed a real issue. I only noticed it after I had already ordered the PCBs, and I really wish I had seen your comment earlier.

It would also be great if the author could provide the original PCB design files instead of just the Gerbers. That way, we could at least make this correction ourselves and generate a revised version of the board if the original design is no longer being updated.
